src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -502,17 +502,16 @@ def run(self, timeout):
                                            [self.context.shared_corpus_path],
                                            self.context.minimized_corpus_path,
                                            self.context.bad_units_path, timeout)
+      symbolized_output = stack_symbolizer.symbolize_stacktrace(result.logs)
```

Comment:
This code is not supposed to exception out, so can keep it at original place, but add return to the logs.log_error branch below (which causes the continue).

Reply:
Isn't it more readable in this form? IMO it's more straightforward to follow right now, but can change if you insist